post-mortems/2018-07-12-discovery-api-delivery-location-patron-lookup-failure.md

```diff
@@ -0,0 +1,32 @@
+# Delivery locations failed to display in Shared Collection Catalog
+
+1. Incident Date
+   - July 12 2018
+2. Incident Summary
+   - Following 'Hold' button for any item in SCC, visitor presented with "Delivery options for this item are currently unavailable..." due to DiscoveryApi failing to look up patron record, which is required to determine visibility of some locations.
+3. Authors
+   - Paul Beaudoin
+   - Stephen Betts
+   - Kevin Friedman 
+4. Timeline
+   - July 11 mid-day: DiscoveryApi deployed to production
+   - July 12 1:51pm: Stephen Betts notes widespread delivery location lookup failures
```

Comment:
Thinks this line should be replaced by (or at least in addition to)
  - July 12 1:34pm: Rosa Caballero-Li alerted scctechgroup@nypl.org about problem; Twana Fluker subsequently escalated to Stephen Betts & Kevin Friedman